src/tests/mam4_test_config.hpp

```diff
@@ -0,0 +1,10 @@
+#ifndef MAM4_TEST_CONFIG_HPP
+#define MAM4_TEST_CONFIG_HPP
+
+// clang-format off
+#define MAM4_TEST_DATA_DIR "/Users/pres026/Research/EAGLES/mam4xx/src/tests/data"
```

Comment:
Is this used? I bet this directory isn't going to be present in most environments.

Reply:
Oh strange, that looks like IDE funny business.

Reply:
Removed.